driver/csiplugin/gpfs.go

```diff
@@ -210,6 +210,8 @@ func (driver *ScaleDriver) SetupScaleDriver(ctx context.Context, name, vendorVer
 	// Adding Capabilities
 	vcam := []csi.VolumeCapability_AccessMode_Mode{
 		csi.VolumeCapability_AccessMode_MULTI_NODE_MULTI_WRITER,
+		csi.VolumeCapability_AccessMode_SINGLE_NODE_MULTI_WRITER,
+		csi.VolumeCapability_AccessMode_SINGLE_NODE_SINGLE_WRITER,
```

Comment:
should we add SINGLE_NODE_WRITER volumecapability too for backward compatibility SINGLE_NODE_MULTI_WRITER & SINGLE_NODE_SINGLE_WRITER ?

Reply:
Done

Reply:
do we also require to add similiar capability in NodeserviceCapability ?

Reply:
no